run.sh

```diff
@@ -34,6 +34,9 @@ source ${scripts_absolute_dir}/deployment-actions-${DEPLOYMENT_MECHANISM}.sh
 deployment_action=${1:-"setup_everything"}
 
 case "$deployment_action" in
+    "deploy_network")
```

Comment:
It's kinda weird to have a new interface for "deploying networks". But I could be good with it. @jgu17 do you see its value on the airship side to have a user interface to deploy networks, or should we hide this?

Reply:
I do not see reason for hiding, some tasks are for developers only anyway